modules/mysql/mycnf.go

```diff
@@ -0,0 +1,77 @@
+package mysql
```

Comment:
Why not use [one of these libraries](https://github.com/search?q=language:Go%20my%20cnf&ref=opensearch)?

Reply:
dunno, i found 

> https://github.com/go-sql-driver/mysql/issues/542

there is this comment in OP

> great job at this is Python's PyMySQL

i decided to check Python's PyMySQL code and implemented something like it does

I saw there are some libraries but all of them not popular (based on stars). Since the thing is not diffucult and basically is just reading/parsing the config in `ini` format i decided to write the code myself.

Reply:
Fair